### PR TITLE
Fix: trigger last price update on ticker and open orders change

### PR DESCRIPTION
**Pull request by josadcha**

### CODE REVIEW

File: src/containers/OrderBook/index.tsx [hunk 1]
```diff
@@ -122,20 +131,18 @@ class OrderBookContainer extends React.Component<Props, State> {
...
     private lastPrice = () => {
         const { marketTickers, currentMarket } = this.props;
-        const defaultTicker = {
-            last: 0,
-            price_change_percent: '+0.00%',
-        };
-        if (currentMarket && marketTickers[currentMarket.id] && marketTickers[currentMarket.id].price_change_percent) {
+        const currentTicker = currentMarket && this.getTickerValue(currentMarket, marketTickers);
+
+        if (currentMarket && currentTicker) {
```
[the remark this answers is not in the document]
Comment (josadcha) [reply src/containers/OrderBook/index.tsx:136]: You're so right

File: src/containers/OrderBook/index.tsx [hunk 2]
```diff
@@ -122,20 +131,18 @@ class OrderBookContainer extends React.Component<Props, State> {
...
           const cn = classNames('', {
-            'cr-combined-order-book__market-negative': (marketTickers[currentMarket.id] || defaultTicker).price_change_percent.includes('-'),
-            'cr-combined-order-book__market-positive': (marketTickers[currentMarket.id] || defaultTicker).price_change_percent.includes('+'),
+            'cr-combined-order-book__market-negative': currentTicker.price_change_percent.includes('-'),
```
Comment (akhlopiachyi) [inline src/containers/OrderBook/index.tsx:138]: fix indentation

**Comment by oyershov** [comment]
Should be fixed by removing shouldComponentUpdate

**Comment by oyershov** [comment]
It`s better to keep shouldComponentUpdate method. It prevents at least 1 re-render every second

File: src/containers/OrderBook/index.tsx [hunk 3]
```diff
@@ -122,23 +132,21 @@ class OrderBookContainer extends React.Component<Props, State> {
...
-                  <span className={cn}>
-                      {Decimal.format(Number((marketTickers[currentMarket.id] || defaultTicker).last), currentMarket.price_precision)} {currentMarket.quote_unit.toUpperCase()}
-                  </span>
-                  <span>{this.props.intl.formatMessage({id: 'page.body.trade.orderbook.lastMarket'})}</span>
-              </React.Fragment>
+        const currentTicker = currentMarket && this.getTickerValue(currentMarket, marketTickers);
+
+        if (currentMarket && currentTicker) {
+            const cn = classNames('', {
+                    'cr-combined-order-book__market-negative': currentTicker.price_change_percent.includes('-'),
```
Comment (akhlopiachyi) [inline src/containers/OrderBook/index.tsx:139]: Please, fix indentation

**Pull request merged**